### PR TITLE
[Feature] Towards game state in firebase database

### PR DESCRIPTION
**Pull request by AmrAnwar**

### CODE REVIEW

**Review by HannesSttzr — DISMISSED** [review]

File: src/main/java/com/chess/backend/repository/Firebase.java [hunk 1]
```diff
@@ -44,8 +73,6 @@ public Firebase() {
         catch(IOException e) {
             System.out.println("ERROR: invalid service account credentials. See README.");
             System.out.println(e.getMessage());
-
-            System.exit(1);
```
Comment (v411e) [inline src/main/java/com/chess/backend/repository/Firebase.java]: Please leave it like it is ;D
It makes no sense if we leave the server running with wrong service account credentials ;)

File: src/main/java/com/chess/backend/restController/controller/JoinOnlineGameController.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+//    @Operation(
+//            summary = "Start a new online game",
+//            description = "Returns the Game ID of a newly created game."
+//    )
+//    @PostMapping
+//    public int joinOnlineGameController(@RequestParam(value = "gameID") int gameID) {
+//          return 1;
+////        return newGameService.getNewGameID(newGameObject);
+//    }
+//}
```
Comment (v411e) [inline src/main/java/com/chess/backend/restController/controller/JoinOnlineGameController.java:40]: Please delete the controller if you do not need it

**Review by v411e — COMMENTED** [review]
There are some comments :)

File: src/main/java/com/chess/backend/repository/Firebase.java (hunk 1 above)
Comment (AmrAnwar) [reply src/main/java/com/chess/backend/repository/Firebase.java]: okidoki

**Comment by AmrAnwar** [comment]
@v411e I'll close this pull request because it's same code in pull request #137

**Pull request closed without merging**